doc/team_practices.md

```diff
@@ -38,24 +42,45 @@ and [view the doc folder](https://github.com/18F/C2/tree/master/doc).
 1. Create a Pull Request. The PR should reference the story URL and include a brief description
 of the changes, including any rationale for how/why the story is addressed in the way it is. Use the format
 `[#STORYID]` or `[Delivers #STORYID]` or `[Fixes #STORYID]` in the PR title. 
-See the [story tracker documentation](https://www.pivotaltracker.com/help/api?version=v5#Tracker_Updates_in_SCM_Post_Commit_Hooks).
+See the [story tracker documentation](https://www.pivotaltracker.com/help/api?version=v5#Tracker_Updates_in_SCM_Post_Commit_Hooks). Consider adding any notes relevant to testing the change, especially for live QA testing.
 
 1. Click **Finish** on the story in the story tracker.
 
-1. Someone from the team should review the PR. If you do not get feedback within 24 hours, assign the PR to a team member.
-The Reviewer is encouraged to indicate a "Ship it" (or your favorite Ship It emoticon) before merging the pull request. 
-Consider pointing out the awesomeness of your teammate's code, too.
-
-1. The Reviewer merges the PR. Pro tip: If the PR title follows the conventions above, 
+1. Someone from the team should review the PR. If you do not get feedback within 24 hours, 
+assign the PR to a team member for review. The Reviewer should consider the automated
+testing status and Code Climate reports, in addition to checking the code for architectural
+consistency, style and legibility. Code reviews are encouraged early in the development process;
+you can always create a PR with a `[WIP]` prefix before you are ready to deliver, and ask for review
+of work-in-progress.
+
+1. The Reviewer should also test the code. This can be done on the `c2-dev` or `c2-staging` environment.
```

Comment:
That's a very valid point, and it bolsters the idea (imo) of having dedicated QA folks who can keep the pipeline moving wrt PRs.

I guess this comes down to risk and mitigation and how we feel about master. I like master HEAD to always be deployable, since it is easiest on the deployer. Our current process makes that much riskier, because there is zero integration or user testing done before merging to master. Process is always about the "appropriate amount of friction", and it's still unclear to me whether the risk associated with an unproven HEAD is acceptable or not, and if not, where we need to add appropriate friction.